dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -32,21 +24,17 @@ COPY \
 {% for whl in docker_platform_monitor_whls.split(' ') -%}
 python-wheels/{{ whl }}{{' '}}
 {%- endfor -%}
-python-wheels/
+/python-wheels/
 {%- endif %}
 
-{% if docker_platform_monitor_whls.strip() %}
-# Install built Python wheels
-RUN pip install \
-{% for whl in docker_platform_monitor_whls.split(' ') -%}
-python-wheels/{{ whl }}{{' '}}
-{%- endfor %}
-{%- endif %}
+# Copy dependencies of sonic-ledd
+COPY ["python-wheels/sonic_platform_common-*-py2-*.whl", "python-wheels/swsssdk-*-py2-*.whl", "/python-wheels/"]
```

Comment:
It is better to specify the dependencies in the .mk file and let the dockerfile to use that dependencies to install necessary packages.